pyproject.toml

```diff
@@ -38,7 +38,7 @@ nautobot = "^1.4.0"
 nautobot-device-lifecycle-mgmt = { version = "^1.0.2", optional = true }
 oauthlib = { version = ">=3.1.0", optional = true }
 packaging = ">=21.3, <24"
-prometheus-client = "~0.14.1"
+prometheus-client = ">=0.14.1"
```

Comment:
not sure why this is relevant in this PR

Reply:
Doesn't need to be, removed it